Address WGLC comments (#57)
ietf-wg-dnsop/draft-ietf-dnsop-structured-dns-error: draft-ietf-dnsop-structured-dns-error.md

server and (re)signs it using the private key from the local root
certificate.

>    * However, this approach is ineffective when DNSSEC is deployed, as DNSSEC

Reviewer: ```suggestion
   * However, this approach is ineffective when DNSSEC is deployed given that DNSSEC
```

     ensures the integrity and authenticity of DNS responses, preventing forged DNS
     responses from being accepted.

   * The HTTPS server will have access to the client's IP address and the hostname being requested.
     This information will be sensitive, as it will expose the user's identity and the

>      domain the user attempted to access.

Reviewer: ```suggestion
     domain name that a user attempted to access.
```

>    * Configuring the local root certificate on endpoints is

Reviewer: ```suggestion
   * Configuring a local root certificate on endpoints is
```

     new local root certificate.

2. The DNS response is forged to provide a NXDOMAIN response to cause
the DNS lookup to terminate in failure. However, this approach does not
work with DNSSEC. In scenarios without DNSSEC, an end user does not know why

> the domain cannot be reached and may repeatedly try to reach the domain

Reviewer: ```suggestion
a domain name cannot be reached and may repeatedly try to reach that domain name
```

> but with no success. Frustrated, the end user may use insecure connections

Reviewer: ```suggestion
but without success. Frustrated, an end user may use insecure connections
```

> to reach the domain, potentially compromising both security and privacy.

Reviewer: ```suggestion
to reach the domain name, potentially compromising both security and privacy.
```

: 'UTF-8'-encoded human-friendly name of the organization that filtered this particular DNS query.
: This field is optional.

l: (language)

> : The "lang" field indicates the language used for the JSON-encoded "j" and "o" fields  and MUST conform to the

Reviewer: ```suggestion
: The "lang" field indicates the language used for the JSON-encoded "j" and "o" fields.  The value of this field MUST conform to the
```

>   language tag syntax specified in {{!RFC5646}}.

Reviewer: ```suggestion
  language tag syntax specified in {{Section 2.1 of !RFC5646}}.
```

  "j": "malware present for 23 days",
  "s": 1,
  "o": "example.net Filtering Service"

>   "l": "en-US",

Reviewer: ```suggestion
  "l": "tzm",
```

Reviewer: for more diversity and localization

| j | justification | UTF-8-encoded {{!RFC5198}} textual justification for a particular DNS filtering | Y | {{name-spec}} of RFCXXXX |
| s | suberror | the suberror code for this particular DNS filtering | N | {{name-spec}} of RFCXXXX |
| o | organization | UTF-8-encoded human-friendly name of the organization that filtered this particular DNS query | N | {{name-spec}} of RFCXXXX |

> | l | language     | Indicates the language of the "j" and "o" fields as defined in RFC 5646 | No | {{name-spec}} of RFCXXXX |

Reviewer: ```suggestion
| l | language     | Indicates the language of the "j" and "o" fields as defined in {{!RFC5646}} | No | {{name-spec}} of RFCXXXX |
```
